packages/react-union/src/components/Union/Union.js

```diff
@@ -19,88 +24,115 @@ class Union extends Component {
 		 */
 		children: PropTypes.node,
 		/**
-		 * Called after the scan of the HTML is done.
+		 * Pre-scanned initial props.
+		 */
+		initialProps: PropTypes.objectOf(PropTypes.object),
+		/**
+		 * Called after the scan of the HTML is successfully done.
 		 */
 		onScanEnd: PropTypes.func,
 		/**
-		 *  Called when there is an error while scanning of the HTML.
+		 * Called when there is an error while scanning of the HTML.
 		 */
 		onScanError: PropTypes.func,
 		/**
-		 * Called before the scan of the HTML
+		 * Called before the scan of the HTML.
 		 */
 		onScanStart: PropTypes.func,
 		/**
-		 * Element in which the scan is running. By default `document`.
+		 * HTML element or Cheerio wrapper in which the scan is running. By default `document`.
 		 */
-		parent: PropTypes.object,
+		parent: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
 		/**
-		 *  Array of routes that are supported by your application.
+		 * Array of routes that are supported by your application.
 		 */
-		routes: PropTypes.arrayOf(PropTypes.shape(RouteShape)).isRequired,
+		routes: PropTypes.arrayOf(PropTypes.shape(RouteShape)),
 		/**
-		 * Enable React.Strict mode. By default `true`
+		 * Enable React.Strict mode. By default `true`.
 		 */
 		strictMode: PropTypes.bool,
```

Comment:
I think, that instead of `strictMode` we should just give an option to user add his own `wrapper` for the widgets... and even this is useless.. user can wrap his components on its own.
I suggest removing this property.

Reply:
Yeah, that's true. It will simplify the component as well, which will help. I'll remove the prop altogether.